tests/e2e/tests/dashboard/netcat-rules.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   ports:
   - port: 44444
-    name: netcat
+    name: tcp-netcat
```

Comment:
why this renaming?

Reply:
It was renamed in order to pass the test. I will revert the change when http inspector comes in.